ui/plugins/controller.js

```diff
@@ -314,6 +314,27 @@ treeherder.controller('PluginCtrl', [
             }
         };
 
+        $scope.backfillJob = function() {
+            if ($scope.user.loggedin) {
+                // Only backfill if we have a valid loaded job, if the user
+                // tries to backfill eg. via shortcut before the load we warn them
+                if ($scope.job.id) {
+                    ThJobModel.backfill($scope.repoName, $scope.job.id).then(function() {
+                        thNotify.send("Initiated request for backfilling this job: " +
+                                       $scope.jobSearchStr, 'success');
+                    }).catch(function(e) {
+                        // Generic error eg. the user doesn't have LDAP access
+                        thNotify.send(
+                            ThModelErrors.format(e, "Unable to send backfill"), 'danger');
```

Comment:
Ed is pretty comfortable with 90-100chars line length, if you desired to move this up a line.